src/containers/InvitationContainer.tsx

```diff
@@ -14,22 +15,23 @@ interface InvitationContainerProps {
 }
 
 function InvitationContainer({ invitationId }: InvitationContainerProps) {
+  const [isLodding, setIsLodding] = useState(true)
   const dispatch = useDispatch()
   const history = useHistory()
   const invitation = useSelector(invitationSelector.getInvitation)
 
   useEffect(() => {
     ;(async () => {
       try {
-        await dispatch(invitationAction.getInvitation({ invitationId })).promise
+        await dispatch(invitationAction.getInvitation({ invitationId })).promise.then(() => setIsLodding(false))
```

Comment:
`isLodding`을 사용해서 로딩과 초대장 페이지의 전환이 일어나도록 함

Reply:
```suggestion
        await dispatch(invitationAction.getInvitation({ invitationId })).promise
        setIsLodding(false)
```
여기서 비동기적 로직을 쓴다면 이미 es2017문법이 쓰였기 때문에 async await문법을 쓰는게 좋을것 같아요
추가적으로 try구문 안에서만 `isLodding`값을 업데이트 시켜준다면 서버에러 등의 이유로 에러가 발생해서 catch구문으로 가게 된다면 `isLodding`이 업데이트 되지 않아 계속 로딩상태의 화면을 보게 될수 있어 finally문 안에서 업데이트 시키는게 좋을것 같아요